ci/taos/config/config-plugins-postbuild.sh

```diff
@@ -76,3 +76,9 @@ echo "[MODULE] plugins-staging: Plugin group that does not have evaluation and a
 # echo "[DEBUG] Current path: $(pwd)."
 # source ${REFERENCE_REPOSITORY}/ci/taos/plugins-staging/${postbuild_plugins[idx]}.sh
 
+
+postbuild_plugins[++idx]="pr-postbuild-build-yocto-kas"
```

Comment:
You need to append annotation (e.g., #) because this module will be applied to the existing TAOS-CI in real-time.
Note that the **pr-postbuild-build-yocto-kas.sh** file will meet errors due to the not-installed packages. 
```bash
# $ sudo apt-get -y install gawk wget git-core diffstat unzip texinfo gcc-multilib
# $ sudo apt-get -y install build-essential chrpath socat libsdl1.2-dev xterm
# $ sudo pip3 install kas

```

Reply:
Thanks! I commented out this new module.